Build error :(

Build on Linux
OS: Gentoo,
ARCH: x86_64,
Kernel: 5.4.143-gentoo-dist.

```
git clone https://github.com/mTvare6/hello-world.rs
cd hello-world.rs
cargo build --release
```

output:
```
   Compiling hello-world v0.1.0 (/home/shiz01/trash/hello-world.rs)
error[E0308]: mismatched types
  --> src/main.rs:14:43
   |
14 | ... 18] = &["en", "es", "bg", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ro", "ru", "sk", "tr", "zh"];
   |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an array with a fixed size of 18 elements, found one with 19 elements

error[E0061]: this function takes 1 argument but 2 arguments were supplied
   --> src/main.rs:117:85
    |
117 | ...                   format!("Oh dear, only {} bytes were written!", french_number(n_bytes, &PRE_REFORM_FEMININE)),
    |                                                                       ^^^^^^^^^^^^^ -------  -------------------- supplied 2 arguments
    |                                                                       |
    |                                                                       expected 1 argument
    |
note: function defined here
   --> /home/shiz01/.cargo/registry/src/github.com-1ecc6299db9ec823/french-numbers-1.1.2/src/lib.rs:321:8
    |
321 | pub fn french_number<N: Integer + FromPrimitive + ToPrimitive + Display + CheckedMul>(
    |        ^^^^^^^^^^^^^

Some errors have detailed explanations: E0061, E0308.
For more information about an error, try `rustc --explain E0061`.
error: could not compile `hello-world` due to 2 previous errors


```

--- a/src/main.rs
+++ b/src/main.rs
@@ -11,7 +11,7 @@ use std::marker::PhantomData;
 use french_numbers::*;
 
     /// These constants are to avoid magic strings/values.
-    const LANGUAGE_LOCALES: &[&str; 18] = &["en", "es", "bg", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ro", "ru", "sk", "tr", "zh"];
+    const LANGUAGE_LOCALES: &[&str; 19] = &["en", "es", "bg", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ro", "ru", "sk", "tr", "zh"];
     const LANGUAGES_DIRECTORY: &str = "translations";
     const MSG: &str = "msg";
 
@@ -114,7 +114,7 @@ fn write(&self, string: &[u8]) -> Result<usize, std::io::Error> {
                         // is the best because not only is it verbose, it also separates what you
                                 // want to format from the template so that you have no idea which argument
                                     // is being inserted where. How thoughtful of Rust!
-                                    format!("Oh dear, only {} bytes were written!", french_number(n_bytes, &PRE_REFORM_FEMININE)),
+                                    format!("Oh dear, only {} bytes were written!", french_number_options(n_bytes, &PRE_REFORM_FEMININE)),
                                 ))
                         } else {
                             // Always flush, especially when you are in public.

